packages/web-vite/src/pages/LabelsPage.tsx

```
@@ -135,156 +171,226 @@ export function LabelsPage() {
 
-      {(showCreateForm || editingLabel) && (
-        <div className="label-form-card">
-          <h2>{editingLabel ? 'Edit Label' : 'Create New Label'}</h2>
-          <form
-            onSubmit={editingLabel ? handleUpdateLabel : handleCreateLabel}
+      {/* Header with search and create button */}
+      <div className="labels-header">
+        <h1 className="labels-title">Labels</h1>
+        <div className="labels-header-actions">
+          <div className="search-box">
+            <svg className="search-icon" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)